apps/aehttp/src/ws_handler.erl

```diff
@@ -43,6 +54,12 @@ websocket_info({gproc_ps_event, Event, Data}, Req, State) ->
 websocket_info({send, SenderName, Action, Payload}, Req, State) ->
     Msg = create_message(SenderName, Action, Payload),
     {reply, {text, jsx:encode(Msg)}, Req, State};
+websocket_info({oracle_query_tx, QTx}, Req, State) ->
+    Msg = create_message_from_event(oracle_query_tx, QTx),
+    {reply, {text, jsx:encode(Msg)}, Req, State};
```

Comment:
Are the outputs (and inputs) of the WS API documented in any schema? It looks like [Swagger has support for WebSockets (ws/wss) both in version 2 (that we use) and version 3](https://github.com/OAI/OpenAPI-Specification/issues/1140).

Reply:
The consensus (at least with input from Michal and Dimitar) seems to be that swagger is the wrong thing to do. I will make a structured documentation of the five "endpoints"/"actions" for Oracle - for now I'll put them in `aeterninty/protocol`